smalld/smalld.py

```diff
@@ -204,10 +206,15 @@ def send_request(self, method, path, payload="", attachments=None):
             args = {"json": payload}
         else:
             args = {}
+        req = requests.Request(method, f"{self.base_url}/{path}", **args)
```

Comment:
So, we have the method and the url, and we pack them into a request so we can pass it to on_request, where we unpack it into method and url?